src/components/OpenLayersMap.js

```diff
@@ -67,7 +67,19 @@ class OpenLayersMap extends React.Component{
         newFeatures.push(feature);
       }
     });
+    hydrants.forEach((hydrant) => {
```

Comment:
Guess we should probably call this something other than renderTrails if it also handles hydrants.

Reply:
Yes, should be renamed.